gradle/build-logic/convention/build.gradle.kts

```diff
@@ -12,6 +13,36 @@ java {
     }
 }
 
+spotless {
+    kotlin {
+        target("src/**/*.kt")
+        ktlint(libs.versions.ktlint.get())
+        licenseHeaderFile(rootProject.file("../../spotless/google-copyright.txt"))
+            .named("google")
+            .onlyIfContentMatches("Copyright \\d+,* Google")
+        licenseHeaderFile(rootProject.file("../../spotless/cb-copyright.txt"))
+            .named("cb-existing")
+            .onlyIfContentMatches("Copyright \\d+,* Christopher Banes")
+        licenseHeaderFile(rootProject.file("../../spotless/cb-copyright.txt"))
+            .named("cb-none")
+            .onlyIfContentMatches("^(?!// Copyright).*\$")
+    }
+
+    kotlinGradle {
+        target("*.kts")
+        ktlint(libs.versions.ktlint.get())
+        licenseHeaderFile(rootProject.file("../../spotless/google-copyright.txt"), "(^(?![\\/ ]\\**).*$)")
+            .named("google")
+            .onlyIfContentMatches("Copyright \\d+,* Google")
+        licenseHeaderFile(rootProject.file("../../spotless/cb-copyright.txt"), "(^(?![\\/ ]\\**).*$)")
+            .named("cb-existing")
+            .onlyIfContentMatches("Copyright \\d+,* Christopher Banes")
+        licenseHeaderFile(rootProject.file("../../spotless/cb-copyright.txt"), "(^(?![\\/ ]\\**).*$)")
+            .named("cb-none")
+            .onlyIfContentMatches("^(?!// Copyright).*\$")
+    }
+}
+
```

Comment:
@chrisbanes I'm not happy about this duplication but I haven't found a good way to fix this yet. Any thoughts?